RELEASE.md

```diff
@@ -0,0 +1,28 @@
+# how to Release
+
```

Comment:
I think in most setups we've the release information directly in the README.md. I worry that people won't look for a RELEASE.md. Could you move the content to the README.md or mention the RELEASE.md in the README.md?

Reply:
ok, will do